### PR TITLE
Marketplace Vision Post

### PR DESCRIPTION
**Pull request by isabellarussell**
All reviewers: feel free to add anything you feel is missing, edit etc. Open to suggestions, but aiming to get it live by EOD tomorrow (Friday 17th August). Cheers

FYI: Cover image on its way.

### CODE REVIEW

File: _posts/2018/08/2018-08-15-marketplace-vision-post.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+**How will Marketplace be run once it launches? How will you distribute apps?**
+
+Marketplace will emerge to the public in a phased approach, with a simpler version released first that will be open to community feedback, as ever.
+The day to day running of Marketplace will be done by its dedicated in-house team who will initially review apps, and our support team will receive training to incorporate marketplace into their remit. All apps will, however, be uploaded with the contact details of their publisher made publicly available, so any app-related support will fall to the app publisher, not Rocket.Chat who will be responsible for support related to the correct functioning of Marketplace as a platform.
+
+Access to apps will be enabled by becoming an admin on a Rocket.Chat server or by creating a developer account directly on Marketplace.
+Upload of apps to the store will be done by distribution from a central Marketplace server.
+Marketplace will be available at www.rocket.chat/apps and we plan to eventually have a dedicated apps.rocket.cha url.
```
[the remark this answers is not in the document]
Comment (isabellarussell) [reply _posts/2018/08/2018-08-15-marketplace-vision-post.md]: Thaaanks! Haven't made it a clickable link yet- I think the plan is to keep more under wraps for now?

**Comment by geekgonecrazy** [comment]
We do have a forum category we could make usage of: https://forums.rocket.chat/c/rocket-chat-apps
As well as a channel: https://open.rocket.chat/channel/rocketchat-apps

Both could definitely be used.  I'd push the forum a bit more though, because I think this type of discussions would fit better in longer discussion form.

**Comment by graywolf336** [comment]
I agree with Aaron, I think the forums should be pushed for these discussions.

**Comment by isabellarussell** [comment]
I've added the Forum link. Cover photo on the way from Thiago. Off to catch my flight! Ciao for now!

**Pull request merged**